### ISSUE TITLE
Scriptable tangle output

### ISSUE THREAD

**Issue by sverhoeven** [entangled/entangled#50]
In my repo I would like to make a git commit hook which will run tangle and stage any written and generated files into the commit. However to know which files to stage I need to parse an output like 
```
• Writing 'src/py/tasks.py'
• Creating 'src/py/templates/form.html'
```

Which is hard to parse, can an option like `--plain` be added to the `tangle` subcommand that will generate easier to parse output for example

```
src/py/tasks.py
src/py/templates/form.html
```

**Comment by jhidding** [comment]
Great idea!

**Comment by jhidding** [comment]
You can ask for machine readable output with the global `-m` flag, this prints
`+ <filename>` for a created file
`~ <filename>` for a modified file
`- <filename>` for a deleted file
